app/scripts/models/reliers/oauth.js

```diff
@@ -28,6 +28,8 @@ define(function (require, exports, module) {
   var SIGNIN_SIGNUP_QUERY_PARAM_SCHEMA = {
     access_type: Vat.accessType().renameTo('accessType'),
     client_id: Vat.clientId().required().renameTo('clientId'),
+    code_challenge: Vat.string().min(43).max(128),
```

Comment:
You can create [custom types in vat.js](https://github.com/mozilla/fxa-content-server/blob/841e1cf692ce73ca6eeeef365d1ab412a4b3a6cf/app/scripts/lib/vat.js#L13) so that these can be used by other modules without defining them multiple times.